Purchases.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -24,8 +24,8 @@
         "repositoryURL": "https://github.com/Quick/Nimble.git",
         "state": {
           "branch": null,
-          "revision": "af1730dde4e6c0d45bf01b99f8a41713ce536790",
-          "version": "9.2.0"
+          "revision": "0fd4ba405699eb8ca939aefa34babf988f4ff9ff",
+          "version": null
```

Comment:
this is the latest main commit for Nimble, which adds matchers for the `Result` type in swift